src/codec/MapPutCodec.ts

```diff
@@ -0,0 +1,47 @@
+/* tslint:disable */
+import ClientMessage = require('../ClientMessage');
+import {BitsUtil} from '../BitsUtil';
+import {SizeUtil} from '../../lib/codec/Utils';
+import {Data} from '../serialization/Data';
+
+var REQUEST_TYPE = 0x0101;
+var RESPONSE_TYPE = 105;
+
+export class MapPutCodec{
+    static calculateSize(name: string, key: Data, value: Data, threadId: number, ttl: number){
+        // Calculates the request payload size
+        var dataSize = 0;
+        dataSize += SizeUtil.getStringSize(name);
```

Comment:
I know this is not related to codec work but can we move `SizeUtil.getStringSize` to `BitsUtil` and change its name to calculateSizeString with this PR.